server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -1,6 +1,6 @@
 CobolLineReaderImpl.incorrectLineFormat=Unexpected indicator area content
 CobolLineReaderImpl.longLineMsg=Source text cannot go past column 80
-cobolParser.missingSqlKeyword=Missing token EXEC or SQL at execSqlStatement
+cobolParser.missingSqlKeyword=Missing token at EXEC SQL statement
```

Comment:
Error occurs only one them (EXEC or SQL) is missing, that is why OR. Same for CICS, but there is no validation so far.

Reply:
Ok, that makes sense, I just don't like the `execSqlStatement` here - it sounds very technical and doesn't really match the COBOL code style. what about changing it to just "Missing token EXEC or SQL"?

Reply:
> token EXE

Yes, looks good.